runners/flink/flink_runner.gradle

```diff
@@ -68,10 +68,15 @@ test {
   if (System.getProperty("beamSurefireArgline")) {
     jvmArgs System.getProperty("beamSurefireArgline")
   }
-}.onlyIf {
-  // TODO Running tests of all Flink versions in parallel can be too harsh on Jenkins memory
-  // Skip 1.6/1.7 tests for now, to avoid "Exit code 137", i.e. Jenkins host killing the Gradle test process
-  project.name != "beam-runners-flink-1.6" && project.name != "beam-runners-flink-1.7"
+  if (project.jenkins.isCIBuild) {
```

Comment:
Wont we have similar issues on dev machine? Or do we expect those to have more resources available? From my point of view, we could enable that serialisation also on 'non ci builds'?

Reply:
I've never had these problems locally. I've turned this off for local execution because it is much slower when you only work on the flink modules.

Reply:
I've removed the Jenkins switch, to make it consistent. Now that we use mostRunAfter, it behaves much nicer on the dev machine.